lib/src/package_analyzer.dart

```diff
@@ -212,8 +212,10 @@ class PackageAnalyzer {
         message = LineSplitter.split(upgrade.stderr as String).first;
       }
 
-      final isUserProblem = message.contains('version solving failed') ||
-          message.contains('Git error.');
+      final isUserProblem = pubspec.hasGitDependency ||
+          message.contains('version solving failed') ||
+          message.contains('Git error.') ||
+          message.contains('Could not find a file named "pubspec.yaml" in');
```

Comment:
The last one could happen if a nested package or something weird didn't contain a `pubspec.yaml`?
Hmm.. I guess it's a user problem that you depended on said package then...

Reply:
Least leave a comment arguing why these error messages are ignored...

Reply:
Removing the last one as it should be covered with the git ones.